x-pack/test/security_solution_api_integration/test_suites/entity_analytics/risk_engine/trial_license_complete_tier/asset_criticality.ts

```diff
@@ -251,7 +251,7 @@ export default ({ getService }: FtrProviderContext) => {
         expect(body.records[0].id_value).to.eql(records[0].id_value);
       });
 
-      it('should return the next 10 asset criticality records if page=2', async () => {
+      it('@skipInServerless should return the next 10 asset criticality records if page=2', async () => {
```

Comment:
This test was not executed in the PR pipelines because it is in the `@skipInServerlessMKI` group. Now that it has been enabled, it is failing and needs to be fixed.

Reply:
Entity Analytics follow-up issue [here](https://github.com/elastic/kibana/issues/189067).